api/src/integrations/epic/epic-orders.js

```diff
@@ -95,6 +96,49 @@ class EpicOrders {
       defaultLog.debug(`Failed to save Epic Order record - error.stack: ${error.stack}`);
     }
   }
+
+  /**
+   * Create the initial default flavour records for the given order record.
+   *
+   * @async
+   * @param {Order} orderRecord NRPTI Order record (required)
+   * @returns {[*]} results of the create operations.
+   * @memberof EpicOrders
+   */
+  async createFlavourRecords(orderRecord) {
+    if (!orderRecord) {
+      throw Error('saveRecord - required record must be non-null.');
+    }
+
+    const flavourRecords = [];
+
+    try {
+      INTEGRATION_SCHEMA.order.flavourSchemas.forEach(async flavourSchema => {
+        const flavourModel = mongoose.model(flavourSchema);
+
+        const flavourAlreadyExists = await flavourModel.exists({
+          _schemaName: flavourSchema,
+          _master: orderRecord._id
+        });
+
```

Comment:
A user could add the flavour record before this completes resulting in two records.

Reply:
> 
> 
> What bug number is this for?

Not for any bug. See my comment below.